### PR TITLE
Added support for setting cbr hostname in environment variables

### LINKED ISSUE #3: Add support for setting custom cbr hostname
In case of working with cbr proxy or replica service it would be great to set it through the enviroment variable.

### PR DESCRIPTION
**Pull request by lamak**
This resolves #3 issue,
added the environment settings for url scheme and hostname, leaving cbr.ru as defaults.

### CODE REVIEW

File: cbrf/const.py [hunk 1]
```diff
@@ -1,14 +1,20 @@
 # -*- coding: utf-8 -*-
+import os
 
 CURRENCY_CODES = {
     # http://www.cbr.ru/scripts/XML_val.asp?d=0
     'USD': 'R01235',
     'EUR': 'R01239',
 }
 
+CBRF_URL_SCHEME = os.getenv('CBRF_URL_SCHEME', 'http')
```
Comment (egregors) [inline cbrf/const.py:10]: SCHEMA should be replased by "PROTOCOl"

File: README.md [hunk 2]
```diff
@@ -26,6 +26,14 @@ Dev version:
     pip install -e .
 ```
 
+## Settings
+For using with your own hostname set environment variables, for example 
+
+```
+export CBRF_URL_SCHEME=http
```
Comment (egregors) [inline README.md:33]: Change "SCHEME" to "PROTOCOL"

**Review by egregors — CHANGES_REQUESTED** [review]
Update "SCHEME" var | env name.

File: cbrf/const.py (hunk 1 above)
Comment (lamak) [reply cbrf/const.py]: Scheme seem to be correct for URI
https://tools.ietf.org/html/rfc3986#section-3.1

**Comment by lamak** [comment]
Scheme seems to be correct https://tools.ietf.org/html/rfc3986#section-3.1

**Review by egregors — APPROVED** [review]

**Pull request merged**